pyaedt/edb_core/edb_data/simulation_configuration.py

```diff
@@ -130,9 +152,28 @@ def airbox_horizontal_extent(self, value):  # pragma: no cover
         if isinstance(value, (int, float)):
             self._airbox_horizontal_extent = value
 
+    @property
+    def use_airbox_horizontal_extent_multiple(self):
+        """Retrieve the air box horizontal extent multiple value.
+
+        Returns
+        -------
+            bool
+            Multiple value, ``True```will use multiple e.g. extent factor, ``False`` will use absolute dimension.
+            Default value is ``True``.
```

Comment:
```suggestion
        """Multiple value for the horizontal extent multiple value of the air box.

        Returns
        -------
            bool
            Multiple value, ``True```will use multiple e.g. extent factor, ``False`` will use absolute dimension.
            Default value is ``True``.
```